### ISSUE TITLE
[suggestion] the ability to drill under bedrock 

### ISSUE THREAD

**Issue by imcasper** [TED-inc/FabricQuarry#20]
Simulate the drilling of ore under the bedrock, virtually outside the game boundary.

Of course, you need to come up with rules for generating blocks. Mark "used areas" somehow.

On the other hand, this is an amazing opportunity for the player:

-A huge number of resources become available (why are we playing techno mods)

-It is possible to rearrange quarry less often (after an endless "minetest", 63 cells is ridiculous)

-The proposal about pipes will be doubly relevant here.

**Comment by imcasper** [comment]
You start with a cheap quarry, (cheap drill, cheap pipes).
Easy to craft, long digging, shallow, requires a lot of energy (per block).

And you gradually improve it, and you get average. Upgrade average to expensive
Difficult to create, digs quickly, digs deep, requires little energy (per block).

Let's assume that all of the above is implemented.
The quarry uses the coolest drill and super expensive pipes:

8 radius 
256 depth
=>
73984 blocks
=>
1156 stacks
42 large chests (cobblestones)

Let's say every 10 block is a valuable ore:

4 large chests (valuable ore)

Even the most expensive components will be justified

**Comment by TED-inc** [comment]
Good idea, but I'm not sure that I will add it. Maybe in future, but I don't want to add something like "super-duper-mega-quarry". Purpose of this mod - is to add smother transition in TechReborn between industrial tier to matter generation and plasma tier
